go.mod

```diff
@@ -61,7 +61,6 @@ require (
 	github.com/libopenstorage/openstorage v1.0.0
 	github.com/lithammer/dedent v1.1.0
 	github.com/lpabon/godbc v0.1.1 // indirect
-	github.com/miekg/dns v1.1.35
```

Comment:
follow the instructions at the top of the file for updating:

```
// Ensure you've carefully read
// https://git.k8s.io/community/contributors/devel/sig-architecture/vendor.md
// Run hack/pin-dependency.sh to change pinned dependency versions.
// Run hack/update-vendor.sh to update go.mod files and the vendor directory.
```

run `hack/lint-dependencies.sh` locally to make sure everything is up to date before committing/pushing

Reply:
thanks ! ok, i just redid the entire vendor of deps and that fixed it